selinux/init.sls

```diff
@@ -2,5 +2,10 @@
 # SELinux contexts
 
 # Contains semanage utility
+{% if grains['os_family'] == 'RedHat' and grains['osmajorrelease']|int <= 7 %}
 policycoreutils-python:
   pkg.installed
+  {% else %}
+policycoreutils-python-utils:
```

Comment:
Same as `ntp` vs `crony` -- let's keep the same state name as before `policycoreutils-python` but have a conditional `- name: ` that sets the name based on the OS

Reply:
This has been impleneted, just not shown here for some reason. I guess this is an old version of the file because it has the comment attached to it?